<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="Dineshs-MacBook-Pro.local" failures="17" tests="32" name="ArrayTests" time="243.049" errors="0" timestamp="2025-05-17T00:14:02 EDT">
  <testcase classname="testClasses.ArrayTests" name="ArraysinPython" time="5.507"/>
  <testcase classname="testClasses.ArrayTests" name="ArraysinPython" time="5.949">
    <failure type="java.lang.AssertionError" message="expected [SyntaxError: bad input on line 1] but found [Alert: SyntaxError: bad input on line 1]">
      <![CDATA[java.lang.AssertionError: expected [SyntaxError: bad input on line 1] but found [Alert: SyntaxError: bad input on line 1]
at testClasses.ArrayTests.ArraysinPython(ArrayTests.java:91)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- ArraysinPython -->
  <testcase classname="testClasses.ArrayTests" name="ArraysinPython1" time="11.14">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testClasses.ArrayTests.ArraysinPython1(ArrayTests.java:102)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- ArraysinPython1 -->
  <testcase classname="testClasses.ArrayTests" name="ArraysinPython1" time="5.946"/>
  <testcase classname="testClasses.ArrayTests" name="ArraysUsingList" time="4.026"/>
  <testcase classname="testClasses.ArrayTests" name="ArraysUsingList" time="6.012">
    <failure type="java.lang.AssertionError" message="expected [SyntaxError: bad input on line 1] but found [Alert: SyntaxError: bad input on line 1]">
      <![CDATA[java.lang.AssertionError: expected [SyntaxError: bad input on line 1] but found [Alert: SyntaxError: bad input on line 1]
at testClasses.ArrayTests.ArraysUsingList(ArrayTests.java:112)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- ArraysUsingList -->
  <testcase classname="testClasses.ArrayTests" name="ArraysUsingList1" time="11.032">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testClasses.ArrayTests.ArraysUsingList1(ArrayTests.java:123)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- ArraysUsingList1 -->
  <testcase classname="testClasses.ArrayTests" name="ArraysUsingList1" time="6.096"/>
  <testcase classname="testClasses.ArrayTests" name="BasicOperationsinLists" time="4.604"/>
  <testcase classname="testClasses.ArrayTests" name="BasicOperationsinLists" time="6.214">
    <failure type="java.lang.AssertionError" message="expected [SyntaxError: bad input on line 1] but found [Alert: SyntaxError: bad input on line 1]">
      <![CDATA[java.lang.AssertionError: expected [SyntaxError: bad input on line 1] but found [Alert: SyntaxError: bad input on line 1]
at testClasses.ArrayTests.BasicOperationsinLists(ArrayTests.java:133)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- BasicOperationsinLists -->
  <testcase classname="testClasses.ArrayTests" name="BasicOperationsinLists1" time="11.119">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testClasses.ArrayTests.BasicOperationsinLists1(ArrayTests.java:144)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- BasicOperationsinLists1 -->
  <testcase classname="testClasses.ArrayTests" name="BasicOperationsinLists1" time="5.967"/>
  <testcase classname="testClasses.ArrayTests" name="ApplicationsofArray" time="4.167"/>
  <testcase classname="testClasses.ArrayTests" name="ApplicationsofArray" time="5.936">
    <failure type="java.lang.AssertionError" message="expected [SyntaxError: bad input on line 1] but found [Alert: SyntaxError: bad input on line 1]">
      <![CDATA[java.lang.AssertionError: expected [SyntaxError: bad input on line 1] but found [Alert: SyntaxError: bad input on line 1]
at testClasses.ArrayTests.ApplicationsofArray(ArrayTests.java:154)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- ApplicationsofArray -->
  <testcase classname="testClasses.ArrayTests" name="ApplicationsofArray1" time="11.071">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testClasses.ArrayTests.ApplicationsofArray1(ArrayTests.java:165)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- ApplicationsofArray1 -->
  <testcase classname="testClasses.ArrayTests" name="ApplicationsofArray1" time="6.165"/>
  <testcase classname="testClasses.ArrayTests" name="SearchTheArray" time="4.255"/>
  <testcase classname="testClasses.ArrayTests" name="SearchTheArray" time="6.164">
    <failure type="java.lang.AssertionError" message="expected [ SyntaxError: bad input on line 1] but found [Alert: SyntaxError: bad input on line 2]">
      <![CDATA[java.lang.AssertionError: expected [ SyntaxError: bad input on line 1] but found [Alert: SyntaxError: bad input on line 2]
at testClasses.ArrayTests.SearchTheArray(ArrayTests.java:195)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- SearchTheArray -->
  <testcase classname="testClasses.ArrayTests" name="SearchTheArray1" time="11.122">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testClasses.ArrayTests.SearchTheArray1(ArrayTests.java:204)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- SearchTheArray1 -->
  <testcase classname="testClasses.ArrayTests" name="SearchTheArray1" time="6.193">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testClasses.ArrayTests.SearchTheArray1(ArrayTests.java:204)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- SearchTheArray1 -->
  <testcase classname="testClasses.ArrayTests" name="MaxConsecutiveOnes" time="4.182"/>
  <testcase classname="testClasses.ArrayTests" name="MaxConsecutiveOnes" time="6.113">
    <failure type="java.lang.AssertionError" message="expected [SyntaxError: bad input on line 1] but found [Alert: SyntaxError: bad input on line 1]">
      <![CDATA[java.lang.AssertionError: expected [SyntaxError: bad input on line 1] but found [Alert: SyntaxError: bad input on line 1]
at testClasses.ArrayTests.MaxConsecutiveOnes(ArrayTests.java:212)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- MaxConsecutiveOnes -->
  <testcase classname="testClasses.ArrayTests" name="MaxConsecutiveOnes1" time="11.389">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testClasses.ArrayTests.MaxConsecutiveOnes1(ArrayTests.java:221)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- MaxConsecutiveOnes1 -->
  <testcase classname="testClasses.ArrayTests" name="MaxConsecutiveOnes1" time="6.098"/>
  <testcase classname="testClasses.ArrayTests" name="EvenNumberOfDigits" time="4.2"/>
  <testcase classname="testClasses.ArrayTests" name="EvenNumberOfDigits" time="6.1">
    <failure type="java.lang.AssertionError" message="expected [SyntaxError: bad input on line 1] but found [Alert: SyntaxError: bad input on line 1]">
      <![CDATA[java.lang.AssertionError: expected [SyntaxError: bad input on line 1] but found [Alert: SyntaxError: bad input on line 1]
at testClasses.ArrayTests.EvenNumberOfDigits(ArrayTests.java:229)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- EvenNumberOfDigits -->
  <testcase classname="testClasses.ArrayTests" name="EvenNumberOfDigits1" time="11.36">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testClasses.ArrayTests.EvenNumberOfDigits1(ArrayTests.java:238)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- EvenNumberOfDigits1 -->
  <testcase classname="testClasses.ArrayTests" name="EvenNumberOfDigits1" time="6.143"/>
  <testcase classname="testClasses.ArrayTests" name="SquaresofaSorted" time="4.188"/>
  <testcase classname="testClasses.ArrayTests" name="SquaresofaSorted" time="6.148">
    <failure type="java.lang.AssertionError" message="expected [SyntaxError: bad input on line 1] but found [Alert: SyntaxError: bad input on line 1]">
      <![CDATA[java.lang.AssertionError: expected [SyntaxError: bad input on line 1] but found [Alert: SyntaxError: bad input on line 1]
at testClasses.ArrayTests.SquaresofaSorted(ArrayTests.java:246)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- SquaresofaSorted -->
  <testcase classname="testClasses.ArrayTests" name="SquaresofaSorted1" time="11.263">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testClasses.ArrayTests.SquaresofaSorted1(ArrayTests.java:255)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- SquaresofaSorted1 -->
  <testcase classname="testClasses.ArrayTests" name="SquaresofaSorted1" time="6.171"/>
  <testcase classname="baseClass.BaseClass" name="@BeforeSuite SetUp" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite SetUp -->
</testsuite> <!-- ArrayTests -->
